src/lib/components/dashboard/credentials/CredentialDetails.svelte

```diff
@@ -51,6 +57,18 @@
       <div class="text-3xl font-semibold w-full text-left ml-2">
         {credential.name}
       </div>
+      <button
+        class="p-2 mr-3 rounded-lg {$showEditCredentialDialog
+          ? 'bg-osvauld-sensitivebgblue'
+          : ''}"
+        on:click={() => {
+          showEditCredentialDialog.set(true);
+          showAddCredentialDrawer.set(true);
+          showCredentialDetailsDrawer.set(false);
```

Comment:
why set to false after edit they should be able to see the refreshed credential right?